build.gradle.kts

```diff
@@ -1,9 +1,13 @@
 import org.jetbrains.kotlin.gradle.dsl.KotlinJvmProjectExtension
+import io.gitlab.arturbosch.detekt.Detekt
+import name.remal.gradle_plugins.dsl.extensions.javaModuleName
 
 plugins {
     kotlin("jvm")
     id("com.lovelysystems.gradle")
     id("com.github.johnrengelman.shadow")
+    id("io.gitlab.arturbosch.detekt") apply false
+    id("name.remal.check-updates")
```

Comment:
where is this plugin used? what is it?

Reply:
was used to create top level :detektAll task, but could be removed